src/AspNet.Security.OAuth.HubSpot/AspNet.Security.OAuth.HubSpot.csproj

```
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <PackageValidationBaselineVersion>6.0.6</PackageValidationBaselineVersion>
```

As this package would ship with 6.0.7, the baseline is that version, not the current latest version which doesn't have HubSpot.

```suggestion
    <PackageValidationBaselineVersion>6.0.7</PackageValidationBaselineVersion>
```